frontend/src/app/articles/[id]/page.tsx

```
@@ -34,7 +34,10 @@ const ArticleDetail = () => {
       date={blog.date}
       title={blog.title}
     >
-      <div dangerouslySetInnerHTML={{ __html: blog.body }} />
+      <div
+        className="w-full prose mb-32 lg:prose-lg max-w-none sm:mb-40"
+        dangerouslySetInnerHTML={{ __html: blog.body }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)